### PR TITLE
Update limits

### PR DESCRIPTION
**Pull request by erlendcognite**

### CODE REVIEW

File: cognite/client/cognite_client.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ class CogniteClient:
 
                 from cognite import CogniteClient
                 client = CogniteClient()
-                res = client.time_series.get_timeseries()
```
Comment (f1cognite) [inline cognite/client/cognite_client.py]: This is a breaking change...? Should we revert this change?

Comment (erlendcognite) [reply cognite/client/cognite_client.py]: it's in a docstring. fixes outdated documentation.

Comment (erlendcognite) [reply cognite/client/cognite_client.py]: see #226
